x-pack/test/security_solution_api_integration/config/services/security_solution_ess_utils.ts

```diff
@@ -5,18 +5,30 @@
  * 2.0.
  */
 
+import { format as formatUrl } from 'url';
+import supertest from 'supertest';
 import { FtrProviderContext } from '../../ftr_provider_context';
 import { SecuritySolutionUtils } from './types';
 
 export function SecuritySolutionESSUtils({
   getService,
 }: FtrProviderContext): SecuritySolutionUtils {
   const config = getService('config');
-  const supertest = getService('supertest');
+  const supertestService = getService('supertest');
```

Comment:
The implication here is similiar to `supertestWithoutAuth` in x-pack/test/api_integration/services/index.ts line 39.


```suggestion
  const supertestWithoutAuth = getService('supertestWithoutAuth');
```